### ISSUE TITLE
`hideDownloadButton` and `hideDarkModeToggle` not working for scalar.config.json

### ISSUE THREAD

**Issue by deepanshu-rajput-git** [scalar/scalar#4190]
### What happens?

In my scalar.config.json, I have configured:
`"hideDarkModeToggle": true,`
`"hideDownloadButton": true,`

But its not getting reflected in the Webpage.

### What did you expect to happen?

The `DarkModeToggle` and `Download API Docs` button should be hidden.

@antlio: Please take a look.
My [scalar.config.json](https://sandbox.scalar.com/e/DYTdQ)

**Comment by antlio** [comment]
hey @deepanshu-rajput-git - thank you for sharing the config, i manage to get the `hideDownloadButton`, is that in the context of the docs.sclar.com?

**Comment by deepanshu-rajput-git** [comment]
Yes, @antlio. I have my github project for Scalar API docs and I want to hide the `Download API docs` and `DarkMode` toggle button from the API docs page.

**Comment by antlio** [comment]
regarding the latter `hideDarkModeToggle` i'm working on a fix!

**Comment by deepanshu-rajput-git** [comment]
Hey, @antlio any updates on this?

**Comment by antlio** [comment]
hey @deepanshu-rajput-git i guess we chat about this through other issues/PR as well - is there any thing i can help with left here?

**Comment by antlio** [comment]
thank you  @deepanshu-rajput-git! feel free to close this one accordingly

**Comment by deepanshu-rajput-git** [comment]
- @antlio : Please take a look https://github.com/scalar/scalar/issues/4354#issuecomment-2586741181 and I am closing this issue.
